LocalstackBackupStoreIT#shouldNotOverwriteCompletedBackup(Backup)[1]

**Summary**

> Try to answer the following as best as possible

- How often does the test fail? Once ?
- Does it block your work? Fails on jenkins
- Do we suspect that it is a real failure? Not sure

**Failures**

> Outline known failure cases, e.g. a failed assertion and its stacktrace obtained from Jenkins

<details><summary>Example assertion failure</summary>
<pre>
AILURE!

[2022-09-26T16:53:14.599Z] java.lang.AssertionError: 

[2022-09-26T16:53:14.599Z] 

[2022-09-26T16:53:14.599Z] Expecting actual throwable to be an instance of:

[2022-09-26T16:53:14.599Z]   java.util.concurrent.ExecutionException

[2022-09-26T16:53:14.599Z] but was:

[2022-09-26T16:53:14.599Z]   java.util.concurrent.TimeoutException

[2022-09-26T16:53:14.599Z] 	at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1960)

[2022-09-26T16:53:14.599Z] 	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2095)

[2022-09-26T16:53:14.599Z] 	at org.assertj.core.internal.Futures.assertFailedWithin(Futures.java:118)

[2022-09-26T16:53:14.599Z] 	...(56 remaining lines not displayed - this can be changed with Assertions.setMaxStackTraceElementsDisplayed)

[2022-09-26T16:53:14.599Z] 	at io.camunda.zeebe.backup.testkit.SavingBackup.shouldNotOverwriteCompletedBackup(SavingBackup.java:78)

[2022-09-26T16:53:14.599Z] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

[2022-09-26T16:53:14.599Z] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)

[2022-09-26T16:53:14.599Z] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)

[2022-09-26T16:53:14.599Z] 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)

[2022-09-26T16:53:14.599Z] 	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:156)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:147)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestTemplateMethod(TimeoutExtension.java:94)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:103)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:92)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:86)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:217)

[2022-09-26T16:53:14.599Z] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:213)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:138)

[2022-09-26T16:53:14.599Z] 	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:68)

[2022-09-26T16:53:14.599Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)

[2022-09-26T16:53:14.599Z] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)

[2022-09-26T16:53:14.599Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.ForkJoinPoolHierarchicalTestExecutorService$ExclusiveTask.compute(ForkJoinPoolHierarchicalTestExecutorService.java:185)

[2022-09-26T16:53:14.600Z] 	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)

[2022-09-26T16:53:14.600Z] 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)

[2022-09-26T16:53:14.600Z] 	at java.base/java.util.concurrent.ForkJoinTask.awaitDone(ForkJoinTask.java:436)

[2022-09-26T16:53:14.600Z] 	at java.base/java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:979)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask$DefaultDynamicTestExecutor.awaitFinished(NodeTestTask.java:236)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:158)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)

[2022-09-26T16:53:14.600Z] 	at org.junit.platform.engine.support.hierarchical.ForkJoinPoolHierarchicalTestExecutorService$ExclusiveTask.compute(ForkJoinPoolHierarchicalTestExecutorService.java:185)

[2022-09-26T16:53:14.600Z] 	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)

[2022-09-26T16:53:14.600Z] 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)

[2022-09-26T16:53:14.600Z] 	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)

[2022-09-26T16:53:14.600Z] 	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)

[2022-09-26T16:53:14.600Z] 	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)

[2022-09-26T16:53:14.600Z] 	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
</pre>
</details>

**Hypotheses**

> List any hypotheses if you have one; can be ommitted

**Logs**

> If possible, provide more context here, e.g. standard output logs, link to build, etc.

<details><summary>Logs</summary>
<pre>
LOGS
</pre>
</details>
https://ci.zeebe.camunda.cloud/blue/organizations/jenkins/camunda%2Fzeebe/detail/main/844/pipeline

--- a/backup-stores/testkit/src/main/java/io/camunda/zeebe/backup/testkit/SavingBackup.java
+++ b/backup-stores/testkit/src/main/java/io/camunda/zeebe/backup/testkit/SavingBackup.java
@@ -74,7 +74,7 @@ public interface SavingBackup {
 
     // then
     assertThat(saveAttempt)
-        .failsWithin(Duration.ofSeconds(1))
+        .failsWithin(Duration.ofMinutes(1))
         .withThrowableOfType(ExecutionException.class);
 
     final var status = getStore().getStatus(backup.id()).join();

